test/test_view_structures_tab.py

```diff
@@ -1,11 +1,15 @@
 import os
+
+import pydicom
 import pytest
 from pathlib import Path
+from PySide6 import QtWidgets, QtCore
 
 from src.Controller.GUIController import MainWindow
 from src.Model.PatientDictContainer import PatientDictContainer
-from src.Model.ROI import get_contour_pixel, calc_roi_polygon
+from src.Model.ROI import get_contour_pixel, calc_roi_polygon, create_initial_rtss_from_ct, create_roi, merge_rtss
```

Comment:
Line too long - PEP 8 compliance (also lines 59, 127-128, 160-163, 170, 177-178, 190, and 209).